Developers/Newcomers_Guide.rst

```diff
@@ -17,6 +17,12 @@ because we care!
 
 **Congratulations!** You are now part of our community.
 
+.. note::
+    
+    The first thing you should do is check other developers' pull requests to get the 
+    hang of things.
+    You should also start working on difficulty/low labelled issues or help us with artwork, tweets, etc.
```

Comment:
this sounds a bit vague, also I'd rather make like a strong recommendation to do this **after** they did the newcomer issue